app/src/main/res/resource/values-de/strings.xml

```diff
@@ -102,7 +104,7 @@
     <string name="feedback_create_file_failed">Dateierstellung fehlgeschlagen.</string>
     <string name="feedback_need_permission">Berechtigung wurde nicht erteilt.</string>
     <string name="feedback_choose_share_app">Foto teilen</string>
-    <string name="feedback_choose_wallpaper_app">Hitergrundbild setzen</string>
```

Comment:
Haha, what did you have in mind, when translating this? :stuck_out_tongue_winking_eye: 

Now, it's okay, of course.